bentoml/types.py

```diff
@@ -123,7 +121,7 @@ class DefaultErrorContext(InferenceContext):
 class InferenceTask(Generic[Input]):
     def __init__(self, data: Input, context: InferenceContext = None):
         self.data = data
-        self.context = context or InferenceContext()
+        self.context: InferenceContext = context or InferenceContext()
```

Comment:
?

Reply:
Alright, that makes sense. I only added the type annotation to this one, because PyCharm wasn't able to infer the type of `inf_task.context`, probably because there is an `or` there.

Reply:
@akainth015 I don't think you should add them all. It looks redundant. How about replace the `context or InferenceContext()` with `context if context else InferenceContext()`